### PR TITLE
feat: Add windows custom package support.

### PR DESCRIPTION
**Pull request by rjaini**
feat: Add windows custom package support.

### CODE REVIEW

File: kubernetes/template/DeploymentTemplates/script.sh [hunk 1]
```diff
@@ -386,6 +387,19 @@ if [ "$CUSTOM_VNET_NAME" != "" ]; then
...
+
+    cat $AZURESTACK_CONFIGURATION | \
+    jq --arg CUSTOM_PACKAGE $WINDOWS_CUSTOM_PACKAGE '.properties.orchestratorProfile.kubernetesConfig += {"customWindowsPackageURL": $CUSTOM_PACKAGE } '  \
+    > $AZURESTACK_CONFIGURATION_TEMP
+
+    validate_and_restore_cluster_definition $AZURESTACK_CONFIGURATION_TEMP $AZURESTACK_CONFIGURATION || exit $ERR_API_MODEL
+
+    log_level -i "Done updating Windows custom package url details."
+fi
 #####################################################################################
```
Comment (jadarsie) [inline kubernetes/template/DeploymentTemplates/script.sh]: nit pick, extra space before ### separator

File: kubernetes/template/DeploymentTemplates/script.sh [hunk 2]
```diff
@@ -386,6 +387,19 @@ if [ "$CUSTOM_VNET_NAME" != "" ]; then
     log_level -i "Done building custom vnet  definition."
 fi
 
+#####################################################################################
+# custom windows package URL
+if [ "$WINDOWS_CUSTOM_PACKAGE" != "" ]; then
+    log_level -i "Adding Windows custom package url details."
```
Comment (jadarsie) [inline kubernetes/template/DeploymentTemplates/script.sh]: NIT, uppercase URL, it is an acronym

File: kubernetes/template/DeploymentTemplates/script.sh (hunk 1 above)
Comment (rjaini) [reply kubernetes/template/DeploymentTemplates/script.sh]: Resolved.

File: kubernetes/template/DeploymentTemplates/script.sh (hunk 2 above)
Comment (rjaini) [reply kubernetes/template/DeploymentTemplates/script.sh]: Resolved.

**Pull request merged**